lib/db/structs.proto

```diff
@@ -28,6 +28,7 @@ message FileInfoTruncated {
     uint32                permissions    = 4;
     int64                 modified_s     = 5;
     int32                 modified_ns    = 11;
+    uint64                modified_by    = 12;
```

Comment:
Are we sure we want this nullable = false?  Don't we want to allow the field to be blank for reverse compatibility or something?